hack/tests/e2e-ansible.sh

```diff
@@ -98,6 +98,24 @@ test_operator() {
         exit 1
     fi
 
+    header_text "Wait for Operator Pod"
+    if ! timeout 60s bash -c -- "until kubectl get pod -l name=memcached-operator; do sleep 1; done"
+    then
+        error_text "FAIL: Operator pod does not exist."
+        operator_logs
+        exit 1
+    fi
+
+    header_text "Ensure no liveness probe fail events"
```

Comment:
```suggestion
    # The following check will verify that has not any event raised with the pod names as Killing
    # because when the probe fails an event to kill the pod will be raised. 
    # Note: We are not calling the /healthz endpoint because it is not accessible public without service and ingress or route setup in the case of OCP config to expose it.   
    header_text "Ensure no liveness probe fail events"
```